{"name": "test_新建项目功能验证[data0]", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.NoSuchDriverException: Message: Unable to obtain driver for chrome; For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors/driver_location", "trace": "self = <selenium.webdriver.common.driver_finder.DriverFinder object at 0x0000020C01519910>\n\n    def _binary_paths(self) -> dict:\n        if self._paths[\"driver_path\"]:\n            return self._paths\n    \n        browser = self._options.capabilities[\"browserName\"]\n        try:\n            path = self._service.path\n            if path:\n                logger.debug(\n                    \"Skipping Selenium Manager; path to %s driver specified in Service class: %s\", browser, path\n                )\n                if not Path(path).is_file():\n                    raise ValueError(f\"The path is not a valid file: {path}\")\n                self._paths[\"driver_path\"] = path\n            else:\n                output = SeleniumManager().binary_paths(self._to_args())\n                if Path(output[\"driver_path\"]).is_file():\n                    self._paths[\"driver_path\"] = output[\"driver_path\"]\n                else:\n                    raise ValueError(f'The driver path is not a valid file: {output[\"driver_path\"]}')\n                if Path(output[\"browser_path\"]).is_file():\n                    self._paths[\"browser_path\"] = output[\"browser_path\"]\n                else:\n>                   raise ValueError(f'The browser path is not a valid file: {output[\"browser_path\"]}')\nE                   ValueError: The browser path is not a valid file:\n\nmyenv\\Lib\\site-packages\\selenium\\webdriver\\common\\driver_finder.py:75: ValueError\n\nThe above exception was the direct cause of the following exception:\n\n>   ???\n\nE:\\ui_framework\\test_cases_actuator\\conftest.py:99: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\nmyenv\\Lib\\site-packages\\selenium\\webdriver\\chrome\\webdriver.py:45: in __init__\n    super().__init__(\nmyenv\\Lib\\site-packages\\selenium\\webdriver\\chromium\\webdriver.py:50: in __init__\n    if finder.get_browser_path():\nmyenv\\Lib\\site-packages\\selenium\\webdriver\\common\\driver_finder.py:47: in get_browser_path\n    return self._binary_paths()[\"browser_path\"]\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n\nself = <selenium.webdriver.common.driver_finder.DriverFinder object at 0x0000020C01519910>\n\n    def _binary_paths(self) -> dict:\n        if self._paths[\"driver_path\"]:\n            return self._paths\n    \n        browser = self._options.capabilities[\"browserName\"]\n        try:\n            path = self._service.path\n            if path:\n                logger.debug(\n                    \"Skipping Selenium Manager; path to %s driver specified in Service class: %s\", browser, path\n                )\n                if not Path(path).is_file():\n                    raise ValueError(f\"The path is not a valid file: {path}\")\n                self._paths[\"driver_path\"] = path\n            else:\n                output = SeleniumManager().binary_paths(self._to_args())\n                if Path(output[\"driver_path\"]).is_file():\n                    self._paths[\"driver_path\"] = output[\"driver_path\"]\n                else:\n                    raise ValueError(f'The driver path is not a valid file: {output[\"driver_path\"]}')\n                if Path(output[\"browser_path\"]).is_file():\n                    self._paths[\"browser_path\"] = output[\"browser_path\"]\n                else:\n                    raise ValueError(f'The browser path is not a valid file: {output[\"browser_path\"]}')\n        except Exception as err:\n            msg = f\"Unable to obtain driver for {browser}\"\n>           raise NoSuchDriverException(msg) from err\nE           selenium.common.exceptions.NoSuchDriverException: Message: Unable to obtain driver for chrome; For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors/driver_location\n\nmyenv\\Lib\\site-packages\\selenium\\webdriver\\common\\driver_finder.py:78: NoSuchDriverException"}, "description": "新建项目流程功能", "parameters": [{"name": "data", "value": "{'用例名称': '新建项目功能验证', '用例步骤': {'进入项目模块': {'点击': {'元素对象': '//*[@id=\"top-menu\"]/ul/li[3]/a'}}, '打开新建项目界面': {'点击': {'元素对象': '//*[@id=\"content\"]/div[1]/a'}}, '输入项目名称': {'发送数据': {'元素对象': '//*[@id=\"project_name\"]', '数据对象': 'qgnyt_var_test_project_01'}}, '输入标识': {'发送数据': {'元素对象': '//*[@id=\"project_identifier\"]', '数据对象': 'qgnyt_var_test_project'}}, '输入项目描述': {'发送数据': {'元素对象': '//*[@id=\"project_description\"]', '数据对象': 'test_project_01项目描述'}}, '创建': {'点击': {'元素对象': '//*[@id=\"new_project\"]/input[3]'}}, '断言元素是否可见': {'断言元素是否可见': {'元素对象': '//*[@id=\"flash_notice\"]'}}, '切换到管理模块界面': {'点击': {'元素对象': '//*[@id=\"top-menu\"]/ul/li[4]/a'}}, '进入到项目界面': {'点击': {'元素对象': '//*[@id=\"admin-menu\"]/ul/li[1]/a'}}, '断言文本是否在页面源码中': {'断言文本是否在页面源码中': {'数据对象': 'qgnyt_var_test_project_01'}}}}"}], "start": 1747741747330, "stop": 1747741747330, "uuid": "d3a3b1e9-0784-4d5b-a3b0-dc598f9f393c", "historyId": "1ff1305a03f154c0bd4256242e0a4a63", "testCaseId": "381676888e06de1aec1176f7fb729ada", "fullName": "test_cases_actuator.test_cases_actuator.TestCase#test_新建项目功能验证", "labels": [{"name": "feature", "value": "项目"}, {"name": "parentSuite", "value": "test_cases_actuator"}, {"name": "suite", "value": "test_cases_actuator"}, {"name": "subSuite", "value": "TestCase"}, {"name": "host", "value": "Marlatinda"}, {"name": "thread", "value": "22612-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_cases_actuator.test_cases_actuator"}]}